cesium/delete.go

```diff
@@ -151,20 +153,20 @@ func (db *DB) removeChannel(ch ChannelKey) error {
 				}
 				otherDB := db.unaryDBs[otherDBKey]
 				if otherDB.Channel.Index == udb.Config.Channel.Key {
-					return errors.New("[cesium] - could not delete index channel with other channels depending on it")
+					return errors.Newf("cannot delete channel [%s]<%d> because it indexes data in channel [%s]<%d>", udb.Channel.Name, udb.Channel.Key, otherDB.Channel.Name, otherDB.Channel.Key)
```

Comment:
It's probably worth adding a stringer for the channel struct as opposed to repeating the `[%s]<%d>` format